company/getAll.js

```diff
@@ -15,7 +15,11 @@ export const main = async () => {
   try {
     do {
       items = await dynamoDbLib.call('scan', params);
-      items.Items.forEach(item => companies.push(item));
+      items.Items.forEach(item => {
+        if (item.isVerified && item.stripeUserId.length > 0) {
+          companies.push(item);
+        }
+      });
```

Comment:
```
companies =  items.Items.filter(item => item.isVerified && item.stripeUserId.length);
```

Reply:
I am not sure why this is here `params.ExclusiveStartKey = items.LastEvaluatedKey;`. That's why I haven't used `filter`. Maybe @cezaryskura can give some input

Reply:
https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Scan.html

Reply:
Pagination I guess. We might need to change it all the places

Reply:
sorry, I missed that one
make sense